assets/src/edit-story/components/canvas/useCanvasSelectionCopyPaste.js

```diff
@@ -132,7 +132,11 @@ function useCanvasSelectionCopyPaste( container ) {
 						evt.preventDefault();
 					}
 				}
-				const { items } = clipboardData;
+				// Can't use deconstructor here.
```

Comment:
Why not? Just curious...

Reply:
If really not possible: can this be detected somehow with a linter or a type system?

Reply:
Can't think of a reason why this would be impossible...

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/210/commits/3057d144d62bfe2510cde738a173f9f1a605017a